src/resources/views/crud/filters/date.blade.php

```diff
@@ -48,12 +55,16 @@ class="nav-item dropdown {{ Request::get($filter->name)?'active':'' }}">
 @push('crud_list_scripts')
 	<!-- include select2 js-->
 	<script src="{{ asset('packages/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js') }}"></script>
+    @if ($filter_language !== 'en')
```

Comment:
@pxpm it looks like bootstrap-datepicker DOES provide translations after all. Maybe it's better to use their translations instead of adding our PHP files for it then. Would be simpler for us too, right?